presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -1335,7 +1342,7 @@ public void testCosineSimilarity()
 
         assertFunction("cosine_similarity(map(array ['a', 'b', 'c'], array [1.0E0, 2.0E0, -1.0E0]), map(array ['c', 'b'], array [1.0E0, 3.0E0]))",
                 DOUBLE,
-                (2 * 3 + (-1) * 1) / (Math.sqrt(1 + 4 + 1) * Math.sqrt(1 + 9)));
+                (2 * 3 + (-1)) / (Math.sqrt(1 + 4 + 1) * Math.sqrt(1 + 9)));
```

Comment:
this looks like an unrelated change.

Reply:
Yes, I saw a warning in IDE that * 1 is not needed.
Decided it is small and harmless enough to add.

Reply:
I think it's written that way to make the formula more clear. So you can understand where the results are coming from